### PR TITLE
login form validation

### PR DESCRIPTION
**Pull request by Oyinbrakemigrace**

### CODE REVIEW

**Comment by Oyinbrakemigrace** [comment]
[Login form validation.webm](https://github.com/annonymousauthority/P2PCoder/assets/117949160/b38e67b6-b753-4da0-b7c1-0b1e7ec98a61)

File: src/components/userLogin_comp.js [hunk 1]
```diff
@@ -48,22 +100,30 @@ function UserLoginComp() {
 
       <div className="space-y-8">
         <div>
-          <label htmlFor="email address">Email Address</label>
+          <label>Email Address</label>
```
Comment (annonymousauthority) [inline src/components/userLogin_comp.js:103]: add the tag for which this label is for, do same for other labels
`htmlFor`

**Review by annonymousauthority — CHANGES_REQUESTED** [review]
Please remove 

-`.env` `package.json` `package-lock.json` from the PR

**Comment by Oyinbrakemigrace** [comment]
[Login form validation.webm](https://github.com/annonymousauthority/P2PCoder/assets/117949160/b3ab8520-5043-4d8b-ac1f-85f3d08184c5)

**Pull request closed without merging**